tsconfig.json

```diff
@@ -6,32 +6,32 @@
       "dom.iterable",
       "esnext"
     ],
+    "allowJs": true,
     "skipLibCheck": true,
     "strict": true,
     "noUncheckedIndexedAccess": true,
     "forceConsistentCasingInFileNames": true,
     "noEmit": true,
+    "incremental": true,
     "esModuleInterop": true,
-    "module": "nodenext",
-    "moduleResolution": "nodenext",
+    "module": "esnext",
+    "moduleResolution": "node",
```

Comment:
Regenerated this file by deleting it and running `next dev`.

I can't remember why I had `nodenext` here before. I think it was for testing TS 5 in PR #781